### ISSUE TITLE
Allow objects to come from stdin

### ISSUE THREAD

**Issue by mkjpryor-stfc** [stackrox/kube-linter#81]
Allow a YAML stream to come from stdin. Many tools use `-` to indicate stdin, e.g.:

```sh
cat objects.yaml | kube-linter lint -
```

Obviously you wouldn't do that specific thing but, combined with #43, it would make the tool much easier to use in an automated pipeline.

**Comment by viswajithiii** [comment]
@mkjpryor-stfc This is currently supported, and should work. We can definitely document it so it's more obvious, though.

**Comment by mkjpryor-stfc** [comment]
@viswajithiii You're right! I thought I had tried it but obviously made a typo or something. My bad.
